### DISCUSSION TITLE
Configuring New Frigate Build with Detector Tensorrt (Nvidia)

### DISCUSSION THREAD

**Discussion by twister36** [blakeblackshear/frigate#9037] [Ask A Question] [answered]
Hoping to get clarification or tell me if I'm doing something wrong. 
I have built a new computer that is running **image: ghcr.io/blakeblackshear/frigate:0.13.0-beta7-tensorrt**.

I have my Nvidia card linked into Docker and get a response from "nvidia-smi" from within the Frigate container:
![image](https://github.com/blakeblackshear/frigate/assets/114322877/0bed2707-de25-4678-a866-a512b778e427)

I also see the Nvidia card listed in Frigate:
![image](https://github.com/blakeblackshear/frigate/assets/114322877/3a5d7c59-a715-4856-99bc-45ed02673d27)

However, if I attempt to change my detector in the config from CPU to TENSORRT I get the following error:
![image](https://github.com/blakeblackshear/frigate/assets/114322877/555cbfed-12a4-42b2-ae78-da5b14f0080b)

I suspecting that I missing the "Generate Models" section listed at https://docs.frigate.video/configuration/detectors
![image](https://github.com/blakeblackshear/frigate/assets/114322877/52e5b02e-5b6a-47be-9243-36cab4218a19)

However, I'm not 100% sure where I run these commands. in WSL?

Environment:
- Windows 11 Pro
- Docker Desktop
- Frigate 13 beta 7

**Comment by NickM-27** [comment, marked as answer]
You are running the beta so should be using the beta docs https://deploy-preview-6262--frigate-docs.netlify.app/configuration/object_detectors#nvidia-tensorrt-detector

**Reply by twister36** [reply]
Thank you Nick!... it's always something stupid/obvious.... for others looking for the same answer, the new version of Frigate automatically creates the models for your GPU. The beta documentation tells you how to link to those automatically created models.

For example:
detectors:
  tensorrt:
    type: tensorrt
    device: 0 #This is the default, select the first GPU

model:
  path: /config/model_cache/tensorrt/yolov7-320.trt
  input_tensor: nchw
  input_pixel_format: rgb
  width: 320
  height: 320
